src/Spring.js

```diff
@@ -454,7 +460,8 @@ export let TransitionSpring = React.createClass({
           currV[key] = mapTree(zero, currVals[key]);
         });
 
-      let frameRate = now ? (Date.now() - now) / 1000 : FRAME_RATE;
+      // TODO: remove, unused.
+      // let frameRate = now ? (Date.now() - now) / 1000 : FRAME_RATE;
```

Comment:
this one is unused, and when removed - `now` var becomes unused too. So removed.